### ISSUE TITLE
[Bug Report][3.0.3] i18n adapter does not install i18n to Vue instance

### ISSUE THREAD

**Issue by jtuchel** [vuetifyjs/vuetify#16222]
### Environment
**Vuetify Version:** 3.0.3
**Vue Version:** 3.2.45
**Browsers:** Firefox 107.0
**OS:** Linux x86_64

### Steps to reproduce
TL;DR see this README file https://github.com/jtuchel/locthing#reproduction

---

- create a new project via `npm create vuetify` ( taken from https://next.vuetifyjs.com/en/getting-started/installation/#installation )
- install the package `vue-i18n` ( taken from https://next.vuetifyjs.com/en/features/internationalization/#vue-i18n )
- install the vite plugin for SFCs `@intlify/unplugin-vue-i18n` ( taken from https://vue-i18n.intlify.dev/guide/advanced/sfc.html#bundling-with-vite )
- change the Vuetify plugin file to

```ts
import '@mdi/font/css/materialdesignicons.css'
import 'vuetify/styles'
import { createVuetify } from 'vuetify'
import { createVueI18nAdapter } from 'vuetify/locale/adapters/vue-i18n'
import { createI18n, useI18n } from 'vue-i18n'

const i18n = new createI18n({
  legacy: false,
  locale: 'en',
  fallbackLocale: 'en'
})

export default createVuetify({
  locale: {
    adapter: createVueI18nAdapter({ i18n, useI18n })
  }
})
```

- use this Vite configuration file

```ts
import vue from '@vitejs/plugin-vue'
import vuetify from 'vite-plugin-vuetify'
import VueI18nPlugin from '@intlify/unplugin-vue-i18n/vite'
import { defineConfig } from 'vite'
import { fileURLToPath, URL } from 'node:url'
import { resolve, dirname } from 'node:path'

export default defineConfig({
  plugins: [
    vue(),
    vuetify({
      autoImport: true,
    }),
    VueI18nPlugin({
      include: resolve(dirname(fileURLToPath(import.meta.url)), './src/**'), // correct path?! search for every <i18n> tag in .vue files...
    }),
  ],
  define: { 'process.env': {} },
  resolve: {
    alias: {
      '@': fileURLToPath(new URL('./src', import.meta.url))
    },
    extensions: [
      '.js',
      '.json',
      '.jsx',
      '.mjs',
      '.ts',
      '.tsx',
      '.vue',
    ],
  },
  server: {
    port: 3000,
  },
})
```

- try out this sample code

```html
<script setup lang="ts">
import { useLocale } from 'vuetify'
import { useI18n } from 'vue-i18n'
const { current } = useLocale()
const { t } = useI18n()
setTimeout(() => {
  current.value = 'de';
}, 3000);
</script>

<template>
  <div>
    <div>content:</div>
    <div>{{ t('title') }}</div>
  </div>
</template>

<i18n>
  {
    "en": { "title": "title | en" },
    "de": { "title": "Titel | de" }
  }
</i18n>
```

### Expected Behavior
I would expect the app to render `title` and change the content to `Titel` after 3 seconds

### Actual Behavior
You will get the error

> Uncaught SyntaxError: Need to install with 'app.use' function

### Reproduction Link
[https://github.com/jtuchel/locthing](https://github.com/jtuchel/locthing)
